recipe/meta.yaml

```diff
@@ -71,5 +81,3 @@ extra:
     - honnibal
     - ines
     - adrianeboyd
-  skip-lints:
-    - cython_needs_compiler
```

Comment:
Why did you delete this when the linter still complains?
(I assume it is a linter feature that it complains that the C++ compiler is not the (expected) C compiler.)

Reply:
It's because my first try was opening the PR against the main branch.

Reply:
Fixed